sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLQuerySuite.scala

```diff
@@ -2558,6 +2558,26 @@ abstract class SQLQuerySuiteBase extends QueryTest with SQLTestUtils with TestHi
     }
   }
 
+  test("SPARK-33630: Support SHOW command as table valued function") {
```

Comment:
Please, place tests like this to the unified test suites for v1/v2 commands. See `ShowTablesSuite`.